lib/trader-sdk/test/indicators/bb-percent-b-mfi.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { describe } from 'mocha';
+import { expect } from 'chai';
+import _ from 'lodash';
+import { FourteenDayPeriodWalksTheLine, FourteenDayPeriod } from '../../mocks/mfi.mock';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/dills122/trader-tools/pullRequest?prid=8102010)
```suggestion
import { FourteenDayPeriodWalksTheLine, FourteenDayPeriod } from "../../mocks/mfi.mock";
```